pkg/commands/docker.go

```diff
@@ -85,14 +84,9 @@ func NewDockerCommand(log *logrus.Entry, osCommand *OSCommand, tr *i18n.Translat
 
 	log.Warn(command)
 
-	err = osCommand.RunCommand(
-		utils.ApplyTemplate(
-			config.UserConfig.CommandTemplates.CheckDockerComposeConfig,
-			dockerCommand.NewCommandObject(CommandObject{}),
-		),
-	)
+	err = osCommand.RunCommand(command)
 	if err != nil {
-		dockerCommand.InDockerComposeProject = false
+		//dockerCommand.InDockerComposeProject = false
```

Comment:
do we still want this commented out?

Reply:
na, it needs some code to detect standalone/compose/swarm and to act slightly differently

I'm thinking that the "projects" pane should become a "stack" pane that shows all the stacks, and lets you drill down into the services in that stack.

It might be good to have some UI/UX guidance :)

Reply:
I think that's reasonable. If you're in a docker swarm context, that means there is no services panel, is that correct? If so, it shouldn't be too tight a squeeze to make that panel the same height as the other side panels. Then it would effectively be just like any other of those panels.

Reply:
sadly, no - swarm stacks are essentially equivalent to compose.yaml files - though you get many of them, and each stack has services, which can be one or more running container

most of my swarm cluesters have a bunch of stacks deployed to them.

even worse, more of my swarms have more than one node, and i'm pondering a UI with prometheus and loki support :D

I'm somewhat distracted by a work issue, which is causing me to write more docker cli plugin code using the some-time to be released Docker 19.03

Reply:
haha sounds intense. In that case, I don't see any issue with just adding an additional panel (or two) if needed